src/data/machines.js

```diff
@@ -145,175 +145,176 @@ export const ephemeralExternalIpAddressPrice = { // per hour in dollars for all
 // It parses the API and outputs the response data.
 export const regionToPrices = [
   {
-    name: 'ASIA-EAST1', monthlyStandardDiskPrice: 0.04, monthlySSDDiskPrice: 0.17,
+    name: 'ASIA-EAST1', monthlyStandardDiskPrice: 0.04, monthlySSDDiskPrice: 0.17, balancedDiskPrice: 0.1,
     n1HourlyGBRamPrice: 0.004906, n1HourlyCpuPrice: 0.036602, preemptibleN1HourlyGBRamPrice: 0.000981, preemptibleN1HourlyCpuPrice: 0.00732,
     t4HourlyPrice: 0.35, p4HourlyPrice: NaN, k80HourlyPrice: 0.49, v100HourlyPrice: 2.48, p100HourlyPrice: 1.6,
     preemptibleT4HourlyPrice: 0.069841, preemptibleP4HourlyPrice: NaN, preemptibleK80HourlyPrice: 0.135,
     preemptibleV100HourlyPrice: 0.74, preemptibleP100HourlyPrice: 0.43
   },
   {
-    name: 'ASIA-EAST2', monthlyStandardDiskPrice: 0.0498, monthlySSDDiskPrice: 0.187,
+    name: 'ASIA-EAST2', monthlyStandardDiskPrice: 0.0498, monthlySSDDiskPrice: 0.187, balancedDiskPrice: 0.11,
```

Comment:
I'm surprised that the balanced disk prices are not the same as as ssd prices given [this](https://github.com/broadinstitute/dsp-scripts/pull/122#discussion_r881731087).

Reply:
Never mind. Got clarification on the thread linked to 😊